### ISSUE TITLE
Windows install related bug

### ISSUE THREAD

**Issue by db4** [janestreet/ppx_inline_test#5]
Generated `ppx_install_test.install` misses `.exe` extension for `ppx` executable so Windows install is broken. I've fixed this modifying `js-utils/install_tags.ml `:
replace
```
let sections =
  [ ("lib",
(*...*)
  ; ("bin",
    [ ("built_exec_ppx", Some "../lib/ppx_inline_test/ppx")
```
with
```
let exe fn = if Sys.win32 then fn ^ ".exe" else fn

let sections =
  [ ("lib",
(*...*)
  ; ("bin",
    [ ("built_exec_ppx", Some (exe "../lib/ppx_inline_test/ppx"))
```
But this problem is common to several modules and probably can be fixed in a more general way.

**Comment by diml** [comment]
Thanks, I included the fix internally. The next minor release will fix this for all js packages

**Comment by diml** [comment]
This is fixed by 113.33.03. Now this script lives in [js-build-tools](https://github.com/janestreet/js-build-tools)
